### PR TITLE
VisibleFields Setting for ReverseManyToManyObjectRelation

### PR DESCRIPTION
**Pull request by blankse**
Visible Fields Feature from #3614 for ReverseManyToManyObjectRelation

### CODE REVIEW

File: models/DataObject/ClassDefinition/Data/AdvancedManyToManyRelation.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
 use Pimcore\Model\Document;
 use Pimcore\Model\Element;
 
-class AdvancedManyToManyRelation extends ManyToManyObjectRelation implements CustomResourcePersistingInterface, QueryResourcePersistenceAwareInterface
```
Comment (brusch) [inline models/DataObject/ClassDefinition/Data/AdvancedManyToManyRelation.php]: That was already fixed in 2f7e18f5e76d31e657a3cb83f39e6b0cff04c1bb 
Can you please take that away? (also for `AdvancedManyToManyObjectRelation` 
Thanks!

Comment (blankse) [reply models/DataObject/ClassDefinition/Data/AdvancedManyToManyRelation.php]: I rebased to master

**Comment by brusch** [comment]
This PR causes the same issues as described in #3737 so we have to solve that first, so that we can use the same approach for reverse many-to-many as well.

**Comment by brusch** [comment]
Closed for now. 
Needs to follow similar approach as used here  #4132 
After this PR was merged we can think about how to implement this for reverse relations.

**Pull request closed without merging**